docs/conf.py

```diff
@@ -0,0 +1,107 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+import os
+
+from recommonmark.transform import AutoStructify
+
+# check if we are running on readthedocs.org
+on_readthedocs = os.environ.get('READTHEDOCS', None) == 'True'
```

Comment:
`"READTHEDOCS" in os.environ`? 😄

Reply:
Yep, see https://docs.readthedocs.io/en/stable/builds.html#build-environment

Reply:
doesn't matter in the end.